examples/chrono_binding.cpp

```diff
@@ -0,0 +1,172 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <cassert>
+#include <string>
+#include <iostream>
+
+#if defined(SQLITE_ORM_THREE_WAY_COMPARISON_SUPPORTED)
+
+///////////////////////////////
+/// sys_days binding as TEXT
+/// ///////////////////////////
+#if defined(SQLITE_ORM_THREE_WAY_COMPARISON_SUPPORTED)
+
+#include <chrono>
+#include <sstream>
+#include <regex>
+#include <format>
+
+inline std::chrono::sys_days today() {
+    const auto today = std::chrono::sys_days{floor<std::chrono::days>(std::chrono::system_clock::now())};
+    return today;
+}
+
+/**
+ *  This is the date we want to map to our sqlite db.
+  *  Let's make it `TEXT`
+  */
+
+//  also we need transform functions to make string from enum..
+inline std::string sysDaysToString(std::chrono::sys_days pt) {
+    auto r = std::format("{:%F}", pt);
+    return r;
+}
+
+constexpr auto null_sys_day = std::chrono::sys_days{std::chrono::days{0}};
+
+/**
+ *  and sys_days from string. This function has nullable result cause
+ *  string can be neither `male` nor `female`. i have taken null_sys_day to
```

Comment:
what is `male` and `female`? Please make `i` capitalized

Reply:
sorry... also changed response to std::optional<>